test/topology/test_topology_remove_decom.py

```diff
@@ -135,3 +135,18 @@ async def test_rebuild_node(manager: ManagerClient, random_tables: RandomTables)
     servers = await manager.running_servers()
     await manager.rebuild_node(servers[0].server_id)
     await check_token_ring_and_group0_consistency(manager)
+
+@pytest.mark.asyncio
+async def test_concurrent_removenode(manager: ManagerClient):
+    [s1, s2, s3] = await manager.servers_add(3)
+
+    await manager.server_stop_gracefully(s3.server_id)
+
+    try:
+        await asyncio.gather(*[manager.remove_node(s1.server_id, s3.server_id), 
+                manager.remove_node(s2.server_id, s3.server_id)])
+    except Exception as e: 
+        print(e)
+        logger.info("exception raised due to concurrent remove node requests")
+    else:
+        assert false
```

Comment:
`false` is not a valid Python keyword. In a sense the behavior would match the requirements (the test would fail) although the error would be extremely obscure... you should simulate lack of error and check what happens.

Also, even if you used the correct keyword (`False`), do you think a vague `AssertionError` is very helpful to whoever encounters it?

Reply:
Raised exception with proper error message